src/components/Wizard/Wizard.test.js

```diff
@@ -42,58 +29,293 @@ test('Wizard loading renders properly', () => {
-            )}
+            <Wizard.ReviewSteps>
+              <Wizard.ReviewStep
+                title="Step 1 Review"
+                collapsed
+                onClick={onStepClicked}
+              >
+                <Wizard.ReviewSubSteps collapsed>
+                  <Wizard.ReviewSubStep
+                    label="Review 1.1"
+                    title="Review 1.1"
+                    collapsed
+                    onClick={onSubStepClicked}
+                  >
+                    <Wizard.ReviewContent collapsed>
+                      <Wizard.ReviewItem>
+                        <span className="wizard-pf-review-item-label">
+                          {'Step 1.1 Label'}:
+                        </span>
+                        <span className="wizard-pf-review-item-value">
+                          Brian Johnson
+                        </span>
+                      </Wizard.ReviewItem>
+                      <Wizard.ReviewItem>
+                        <span className="wizard-pf-review-item-label">
+                          {'Step 1.2 Label'}:
+                        </span>
+                        <span className="wizard-pf-review-item-value">
+                          This is the description.
+                        </span>
+                      </Wizard.ReviewItem>
+                    </Wizard.ReviewContent>
+                  </Wizard.ReviewSubStep>
+                </Wizard.ReviewSubSteps>
+              </Wizard.ReviewStep>
+            </Wizard.ReviewSteps>
           </Wizard.Main>
         </Wizard.Row>
       </Wizard.Body>
-      <Wizard.Footer>
-        <Button bsStyle="default" className="btn-cancel">
-          Cancel
-        </Button>
-        <Button bsStyle="default" onClick={onBackButtonClick}>
-          <span className="i fa fa-angle-left" />Back
-        </Button>
-
-        <Button bsStyle="primary" onClick={onNextButtonClick}>
-          Next<span className="i fa fa-angle-right" />
-        </Button>
-      </Wizard.Footer>
     </Wizard>
   );
 
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
+  expect(component.render()).toMatchSnapshot();
+});
+
+const testWizardPattern = props => {
+  const onHide = jest.fn();
+  const onExited = jest.fn();
+  const onStepChanged = jest.fn();
+  return (
+    <Wizard.Pattern
+      show
+      onHide={onHide}
+      onExited={onExited}
+      title="Wizard Pattern Example"
+      shouldDisableNextStep={false}
+      steps={[
+        { title: 'General', render: () => <p>General</p> },
+        { title: 'Step Two', render: () => <p>Step Two</p> },
+        { title: 'Step Three', render: () => <p>Step Three</p> }
+      ]}
+      loadingTitle="Loading..."
+      loadingMessage="This may take a minute."
+      activeStepIndex={0}
+      onStepChanged={onStepChanged}
+      {...props}
+    />
+  );
+};
+
+test('Wizard Pattern renders properly while loading', () => {
+  const component = mount(testWizardPattern({ loading: true }));
```

Comment:
shallow would be a great candidate for these.  mount should really only be used if you need to have access to refs. enzyme has an option for lifecycle as well if the lifecycle is needed in shallow.

Reply:
> enzyme has an option for lifecycle as well if the lifecycle is needed in shallow

enzyme's shallow rendering now supports all React component lifecycle methods - [see this PR for details](https://github.com/airbnb/enzyme/pull/318) which made its way into enzyme [2.4.0 release](https://github.com/airbnb/enzyme/blob/master/CHANGELOG.md#240-july-7-2016).